meson.build

```diff
@@ -356,13 +356,10 @@ else
 endif
 
 if want_libfuzzer
-        fuzzing_engine = meson.get_compiler('cpp').find_library('Fuzzer', required : false)
-        if fuzzing_engine.found()
-                userspace_c_args += '-fsanitize-coverage=trace-pc-guard,trace-cmp'
-        elif cc.has_argument('-fsanitize=fuzzer-no-link')
-                userspace_c_args += '-fsanitize=fuzzer-no-link'
+        if cc.has_argument('-fsanitize=fuzzer-no-link')
+                userspace_c_args += f'-fsanitize=fuzzer-no-link'
```

Comment:
Does meson support `f'...'` ? Why is it necessary here?

Reply:
Oversight, not required